tests/acceptance/features/webUIFilesDetails/fileDetails.feature

```diff
@@ -62,21 +62,19 @@ Feature: User can open the details panel for any file or folder
     When the user switches to "links" accordion item in details panel using the webUI
     Then the "links" details panel should be visible
 
-  @skip @yetToImplement @comments-app-required @public_link_share-feature-required
+  @comments-app-required @public_link_share-feature-required
   Scenario: user shares a file through public link and then the details dialog should work in a Shared by link page
```

Comment:
this test does not create a public link anymore

Reply:
The scenarios need to create public link share so I have used the steps to create public link share.